pkg/virtctl/templates/templates.go

```diff
@@ -58,3 +64,15 @@ func ExactArgs(nameOfCommand string, n int) cobra.PositionalArgs {
 		return nil
 	}
 }
+
+// WarnPaused prints warning message if VMI is paused
```

Comment:
I would avoid the comment and instead put all the information in the function name, e.g. maybe change `WarnPaused` to something like `PrintWarnningForPausedVMI`?

Reply:
Changed the function name. Kept the comment as doc comment for exported function.